packages/plugin-phone/test/integration/spec/call.js

```diff
@@ -223,47 +336,183 @@ describe(`plugin-phone`, function() {
+        it(`is false`, () => {
+          call = spock.spark.phone.dial(mccoy.email, {constraints: {video: false}});
+          return Promise.all([
+            mccoy.spark.phone.when(`call:incoming`)
+              .then(([c]) => c.answer()),
+            call.when(`connected`)
+              .then(() => assert.equal(call.status, `connected`))
+              .then(() => assert.isFalse(call.sendingVideo))
+          ]);
+        });
       });
     });
 
     describe(`#receivingAudio`, () => {
+      let call;
+
+      afterEach(() => {
+        const c = call;
+        call = undefined;
+        return c.hangup().catch((reason) => console.warn(reason));
+      });
+
       describe(`when the local party is receiving Audio`, () => {
-        it(`is true`);
+        it(`is true`, () => {
+          call = spock.spark.phone.dial(mccoy.email, {constraints: {audio: true}});
+          return Promise.all([
+            mccoy.spark.phone.when(`call:incoming`)
+              .then(([c]) => c.answer()),
+            call.when(`connected`)
+              .then(() => assert.equal(call.status, `connected`))
+              .then(() => assert.isTrue(call.receivingAudio))
+          ]);
+        });
       });
 
       describe(`when the local party is not receiving Audio`, () => {
-        it(`is false`);
+        it(`is false`, () => {
+          call = spock.spark.phone.dial(mccoy.email, {constraints: {audio: false}});
+          return Promise.all([
+            mccoy.spark.phone.when(`call:incoming`)
+              .then(([c]) => c.answer()),
+            call.when(`connected`)
+              .then(() => assert.equal(call.status, `connected`))
+              .then(() => assert.isFalse(call.receivingAudio))
+          ]);
+        });
       });
     });
 
     describe(`#receivingVideo`, () => {
+      let call;
+      beforeEach(() => {
+        call = spock.spark.phone.dial(mccoy.email);
+        return Promise.all([
+          mccoy.spark.phone.when(`call:incoming`)
+            .then(([c]) => c.answer()),
+          call.when(`connected`)
+            .then(() => assert.equal(call.status, `connected`))
+        ]);
+      });
+
+      afterEach(() => {
+        const c = call;
+        call = undefined;
+        return c.hangup().catch((reason) => console.warn(reason));
+      });
+
       describe(`when the local party is receiving Video`, () => {
-        it(`is true`);
+        it(`is true`, () => {
+          call = spock.spark.phone.dial(mccoy.email, {constraints: {video: true}});
+          return Promise.all([
+            mccoy.spark.phone.when(`call:incoming`)
+              .then(([c]) => c.answer()),
+            call.when(`connected`)
+              .then(() => assert.equal(call.status, `connected`))
+              .then(() => assert.isTrue(call.receivingVideo))
+          ]);
+        });
       });
 
       describe(`when the local party is not receiving Video`, () => {
-        it(`is false`);
+        it(`is false`, () => {
```

Comment:
Seems like a double negative here. Should this be a part of `when the local party is receiving Video` and `it('is false')`?